src/basic/selinux-util.c

```diff
@@ -507,7 +573,7 @@ int mac_selinux_bind(int fd, const struct sockaddr *addr, socklen_t addrlen) {
         path = strndupa(un->sun_path, addrlen - offsetof(struct sockaddr_un, sun_path));
 
         /* Check for policy reload so 'label_hnd' is kept up-to-date by callbacks */
-        (void) avc_netlink_check_nb();
+        mac_selinux_check_reload_and_update();
```

Comment:
we'd usually call this m̀ac_selinux_maybe_reload()` or so

Reply:
renamed